doc/layer2.md

```diff
@@ -0,0 +1,40 @@
+# Porter Layer2 Usage
+Porter now support bgp and layer2, and will support non-bgp switch in future. In
+order to distinguish them, file `protocol` was added. 
+
+An example of how to use it will follow.
```

Comment:
@duanjiong  Is there any prerequisites when using Layer2 with Porter?

Reply:
There is a config about kube-proxy,  i will doc about it later.